ImageInLib/ImageInLib/src/filter_params.h

```diff
@@ -6,17 +6,17 @@ extern "C" {
 #pragma once
 	typedef struct
 	{
-		dataType timeStepSize;
-		dataType h;
-		dataType sigma;
-		dataType edge_detector_coefficient;
-		dataType omega_c;
-		dataType tolerance;
-		dataType eps2;
+		size_t timeStepSize;
+		double h;
+		double sigma;
+		double edge_detector_coefficient;
+		double omega_c;
+		double tolerance;
+		double eps2;
+		dataType coef;
```

Comment:
Please, why, you changed from dataType=float to double for the other parameters except for "coef"?